templates/quickstart/TMQuickStart.template

```diff
@@ -0,0 +1,419 @@
+      subnet contained the in VPC chosen above.
+    Type: 'AWS::EC2::Subnet::Id'
+    ConstraintDescription: >-
+      RDS Subnet Groups must be comprised of 2 subnets in seperate availability
+      zones within the specified VPC for deploying this template
+  DeepSecuritySubnet:
+    Description: >-
+      Select an existing Subnet for Deep Security Manager. Must be a public
+      subnet contained the in VPC chosen above.
+    Type: 'AWS::EC2::Subnet::Id'
+    MinLength: '1'
+    MaxLength: '255'
+    AllowedPattern: '[-_a-zA-Z0-9]*'
+    ConstraintDescription: Subnet ID must exist in the chosen VPC
+  DeepSecurityAdminName:
+    Default: MasterAdmin
+    Description: The Deep Security Manager administrator username for Web Console Access.
+    Type: String
+    MinLength: 1
+    MaxLength: 16
+    AllowedPattern: '[a-zA-Z][a-zA-Z0-9]*'
+    ConstraintDescription: >-
+      Must begin with a letter and contain only alphanumeric characters. Min
+      length 1, max length 16
+  DeepSecurityAdminPass:
+    NoEcho: true
+    Description: >-
+      The Deep Security Manager administrator password. Must be 8-41 characters
+      long and can only contain alphanumeric characters or the following special
+      characters !^*-_+
+    Type: String
+    MinLength: 8
+    MaxLength: 41
+    AllowedPattern: '[a-zA-Z0-9!^*\-_+]*'
+    ConstraintDescription: >-
+      Can only contain alphanumeric characters or the following special
+      characters !^*-_+ Min length 8, max length 41
+  ProtectedInstances:
+    Description: Select how many instances would you like to protect.
+    Type: String
+    AllowedValues:
+      - 1-100
+      - 101-500
+      - 501-1000
+      - 1001-2000
+  QSS3BucketName:
+    AllowedPattern: ^[0-9a-zA-Z]+([0-9a-zA-Z-.]*[0-9a-zA-Z])*$
+    ConstraintDescription: Quick Start bucket name can include numbers, lowercase
+      letters, uppercase letters, periods (.), and hyphens (-). It cannot start or
+      end with a hyphen (-).
+    Default: trend-micro-quick-start
+    Description: S3 bucket name for the Quick Start assets. Quick Start bucket name
+      can include numbers, lowercase letters, uppercase letters, and hyphens (-).
+      It cannot start or end with a hyphen (-).
+    Type: String
+  QSS3KeyPrefix:
+    AllowedPattern: ^[0-9a-zA-Z-.]+(/[0-9a-zA-Z-.]+)*$
+    ConstraintDescription: Quick Start key prefix can include numbers, lowercase letters,
+      uppercase letters, periods (.), hyphens (-), and forward slash (/). It cannot start or end
+      with forward slash (/) because they are automatically appended.
+    Default: latest
+    Description: S3 key prefix for the Quick Start assets. Quick Start key prefix
+      can include numbers, lowercase letters, uppercase letters, periods (.), hyphens (-), and
+      forward slash (/). It cannot start or end with forward slash (/) because they
+      are automatically appended.
+    Type: String
+Mappings:
+  AWSInfoRegionMap:
```

Comment:
This is no longer used in the Quick Start standards. The URL is dynamically built using Fn::Sub. See other comment around this.